app/Http/Requests/StoreFlightRequest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class StoreFlightRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'airline_id' => ['required', 'integer'],
+            'origin_city_id' => ['required', 'integer'],
+            'destination_city_id' => ['required', 'integer', Rule::notIn([$this->input('origin_city_id')])],
+            'departure_at' => ['required', 'date'],
```

Comment:
```suggestion
            'departure_at' => ['required', 'date_format:[format]'],
```
You should validate that the time of departure, as well as the time of arrival, are specified, otherwise there's no way to know exactly when a flight is supposed to leave or arrive. Replace `[format]` with the corresponding format. You can check if any of [these](https://www.php.net/manual/en/class.datetime.php) constants suit the format you need. Example usage:
```
"date_format:{DateTime::ATOM}"
```
Do this for the `arrival_at` field as well.